chapters/synchronous.tex

```diff
@@ -1083,6 +1092,13 @@ \subsection{Clocked Discrete-Time and Clocked Discretized Continuous-Time Partit
 Such a partition has to be solved with a \emph{solver method} of \cref{solver-methods}.
 When \lstinline!previous(x)! is used on a continuous-time state variable \lstinline!x!, then \lstinline!previous(x)! uses the start value of \lstinline!x! as value for the first clock tick.
 
+The use of the operator sample from \cref{event-related-operators-with-function-syntax} in a clocked partition is problematic.
+Diagnostics are appropriate especially if it is intended to generate sampling faster than the clocks, and otherwise the sampling should ideally be adjusted to the clock ticks.
+\begin{nonnormative}
+Since the goal is that it should be possible to use \emph{every} continuous-time Modelica model, it is not forbidden.
```

Comment:
It may not be obvious what kind of use this refers to;
```suggestion
Since the goal is that it should be possible to use \emph{every} continuous-time Modelica model synchronously, it is not forbidden.
```

I'm afraid that this way of using _synchronously_ could be new, and maybe we should try to stick with some other formation that is already in use?

Reply:
Reused 'sampled data control system' from section introduction.

Reply:
Obsolete after breaking out this part as a separate PR.  Resolving.